doc/docusaurus/docs/essential-concepts/versions.md

```diff
@@ -0,0 +1,68 @@
+You simply write your code in Plutus Tx, compile it to UPLC, and run it with a UPLC evaluator.
+This is what we mean by "they are not really different versions of a language in the sense of programming languages".
+
+It is only when you write a Plutus _validator script_, put it in a transaction and submit it to the Cardano ledger, that ledger language version becomes relevant. It is relevant in three ways:
+
+- Depending on the ledger language version, the ledger will pass different arguments to the validator.
+  For instance, the arguments passed to a Plutus V2 validator include a list of reference inputs of the transaction. This is not passed to a Plutus V1 validator since reference inputs were introduced later than Plutus V1 (specifically, Plutus V1 was introduced in the Alonzo era, while reference inputs were introduced in the Babbage era).
+  Similarly, only Plutus V3 can be used to validate governance actions, since governance actions are introduced in the Conway era and are not part of the pre-Conway era transactions, and thus aren't part of the arguments passed to Plutus V1 or V2 validators.
+- At present, a newer ledger language version has access to more builtin functions than an older ledger language version.
+  Bear in mind that this will change as we plan to make all builtin functions available to all ledger language versions.
+- At present, Plutus Core version (explained below) 1.1.0 can only be used with Plutus V3, while Plutus V1 and V2 validators must use Plutus Core 1.0.0.
+  This will also change, as we plan to make all Plutus Core versions compatible with all ledger language versions.
+
+When do we introduce a new ledger language version?
+A new ledger language version must be introduced when a new ledger era is rolled out, because in a new ledger era, transactions will be different and contain new and/or modified fields, such as governance actions introduced in the Conway era.
+As a result, new information will need to be passed as arguments to Plutus validators.
+This necessitates a new ledger language version, because the arguments passed to existing ledger language versions must stay exactly the same, so that existing validators continue to work.
+
+Previously it was also the case that introducing new builtin functions or new Plutus Core versions necessitate a new ledger language version, but this is no longer the case.
+Introducing a new ledger language version is very expensive as it must be maintained forever.
+Going forward we expect the launch of new ledger eras to be the only reason for introducing new ledger language versions.
+
+## Plutus Core Version
+
+Plutus Core version is the usual sense of version pertaining to programming languages - in this instance the Plutus Core language.
+So far there have been two Plutus Core versions: 1.0.0 and 1.1.0.
```

Comment:
Please let's call it AST version. Having "Plutus Core version" and "`plutus-core` version" to mean different things is kind of horrible.

Reply:
I think AST version is fine, but it is unusual - it is common for a programming language to have a version - e.g. Python 3.7

Reply:
Python 3.7 corresponds to `plutus-core-1.32`, not `1.1.0`.